Ranker.py

```diff
@@ -1,21 +0,0 @@
-import pandas as pd
-import numpy as np
-
-def create_rank(medical_headers, medical_service_lines, icd9_proc, colonoscopy_codes, resection_codes):
```

Comment:
i'm guessing this is the "meat" of the code. consider adding a docstring explaining in words how the ranker works. it's kind of hard to read.